src/main/java/hudson/plugins/analysis/core/ResultAction.java

```diff
@@ -34,7 +34,7 @@
      *
      * @return the associated build of this action
      */
-    AbstractBuild<?, ?> getBuild();
+    Run<?, ?> getBuild();

```

Comment:
Hopefully it is not a binary compatibility issue. Worst case a recompile of the original source of the downstream plugin should fix any such issues though.